docs/mkdocs/secureboot-signing.md

```diff
@@ -0,0 +1,75 @@
+# Signing kernel modules using KMM
+
+### Secureboot with KMM
+
+For more details on using Secureboot see [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_monitoring_and_updating_the_kernel/signing-kernel-modules-for-secure-boot_managing-monitoring-and-updating-the-kernel) or [here](https://wiki.debian.org/SecureBoot)
+
+### Using Signing with KMM
+
+On a secureboot enabled system all kernel modules (kmods) must be signed with a public/private keypair enrolled into the Machine Owner's Key or MOK database. Drivers distributed as part of a distribution should already be signed by the distributions private key, but for kernel modules build out-of-tree KMM supports signing kernel modules using the "sign" section of the kernel mapping.
+
+To use this functionality you need:
+
+ * A public private key pair in the correct (der) format
+ * At least one secureboot enabled nodes with the public key enrolled in its MOK database
+ * A driver container image 
+
+The yaml below will add the public/private keypair as secrets with the required key names ("key" for the private key, "cert" for the public key). It will then pull down the `unsignedImage` image, open it up, sign the kernel modules listed in `filesToSign`, add them back anbd push the resulting image as `containerImage`. 
```

Comment:
Will it just add `-singed` at the end of the image name?

Reply:
We are not adding anything to the name, it is simply consuming `unsignedImage` and producing`containerImage` whatever those are defined to be.
The place where we do append things to names is if we are building in cluster then signing, when we need the build to produce a intermediate image, and signing produces the final image.  In that case the built image is `containerImage-kmm-<modulename>-unsigned`,  and then signing can produce `containerImage`
This example doesn't get into that because its only doing signing, but we probably do need a build & sign example as well that explains all that (and better than I have here!)

Reply:
Ack. Thanks.